.github/workflows/pr-dependabot.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           script: |
             const path = require('path');
             const scriptPath = path.resolve('./.github/workflows/get-pr-info.js');
-            const commitHash = ${{ github.event.workflow_run.head_sha }};
+            const commitHash = `${{ github.event.workflow_run.head_sha }}`
```

Comment:
Should use [SINGLE QUOTE](https://github.com/marilyn79218/react-lazy-show/pull/55/files).